include/libtorrent/aux_/session_impl.hpp

```diff
@@ -352,10 +352,6 @@ namespace aux {
 #ifndef TORRENT_NO_DEPRECATE
 			//deprecated in 1.2
 
-			TORRENT_DEPRECATED
-			void set_load_function(user_load_function_t fun)
-			{ m_user_load_torrent = fun; }
```

Comment:
the reason I left this here is that the client may store handles in the function object, and rely on them being held alive. Perhaps this is a bit far fetched, but it could break some code.

Reply:
handles, like a ``shared_ptr<>`` for instance

Reply:
now I remember, you mentioned this before